test/.setup.js

```diff
@@ -0,0 +1,24 @@
+// Global setup for all tests
+
+// We need jsdom for enzyme mount()'ed components
+// see also: https://github.com/airbnb/enzyme/blob/master/docs/guides/jsdom.md#using-enzyme-with-jsdom
+// see also: https://github.com/lelandrichardson/enzyme-example-mocha/blob/master/test/.setup.js
+
+/*
+var jsdom = require('jsdom').jsdom;
+
+var exposedProperties = ['window', 'navigator', 'document'];
+
+global.document = jsdom('');
+global.window = document.defaultView;
+Object.keys(document.defaultView).forEach((property) => {
+  if (typeof global[property] === 'undefined') {
+    exposedProperties.push(property);
+    global[property] = document.defaultView[property];
+  }
+});
+
+global.navigator = {
+  userAgent: 'node.js'
+};
+*/
```

Comment:
We'll need it the first time any React components are tested, and it's handy for setting up globals like `window` when they can't be avoided by doing a dependency injection thing